source/targetLogic.test.ts

```diff
@@ -55,7 +55,7 @@ describe("getActionForMessage", async () => {
     ["contentScript", "somePage", "frame", "respond"], // Wrong, but won't happen
 
     ["contentScript", "thisTab", "background", "forward"],
-    ["contentScript", "thisTab", "somePage", "forward"],
+    ["contentScript", "thisTab", "somePage", "ignore"],
```

Comment:
The [new tests](https://github.com/pixiebrix/webext-messenger/pull/220) make this change immediately verifiable 🙌